dpos/p2p/msg/proposal.go

```diff
@@ -6,7 +6,7 @@ import (
 	"github.com/elastos/Elastos.ELA/core"
 )
 
-const DefaultProposalMessageDataSize = 164 //67+32+65
+const DefaultProposalMessageDataSize = 134 //33+32+4+65
```

Comment:
67 + 32 + 4 + 65  sponsor is string, length is 66  writvarstring need + 1 byte = 67